src/main/steps/applicant2/check-contact-details/content.ts

```diff
@@ -0,0 +1,37 @@
+import { ApplicationType } from '../../../app/case/definition';
+import { TranslationFn } from '../../../app/controller/GetController';
+
+const en = () => ({
+  title: 'Review your contact details',
+  yourAddress: 'Your address',
+  yourPhoneNumber: 'Your phone number',
+});
+
+// @TODO translations
+const cy = en;
+
+const languages = {
+  en,
+  cy,
+};
+
+export const generateContent: TranslationFn = content => {
+  const address = [content.formState?.applicant2Address1];
+  address.push(content.formState?.applicant2Address2);
```

Comment:
see previous comment

Reply:
same as before, we are passing partial case so its complaining it might be undefined.